cyclops-ui/src/components/pages/EditModule/custom.css

```diff
@@ -26,3 +26,7 @@ code {
     padding: 0;
   }
 }
+
+a:hover {
+  text-decoration: underline;
+}
```

Comment:
This will make all `<a>` be underlined when hovered. Can you make a separate class and add text decoration to it?